### PR TITLE
Add opacity to text and background colors

### LINKED ISSUE #3: Processor for opacity
## Description
We'll want to change the opacity of some elements or paragraphs, and we should be able to do this from Gutenberg.

At this moment, we are planning to do this adding classes to each element, for both the text and the background, and process them from Frontity. Something like `has-text-opacity-80` or `has-background-opacity-80`.

## How it works
* If an element has the class `has-text-opacity-XX`, we should get the opacity number and change the `color` css property. We should create a function to convert to rgba (adding the opacity) inside the `libraries` of the theme. We can get the color of the element as explained [here](https://github.com/frontity/frontity.org/issues/2) or get `primary` as default one.

* If an element has the class `has-background-opacity-XX`, we should get the opacity number and change the `background-color` css property. We can get the background color of the element as explained [here](https://github.com/frontity/frontity.org/issues/2).

* We should be able to use this inside React too, without classes. If we want to apply an opacity to one of the `state.theme.colors` from React, we should be able to do it.

## Where to check
* Inside `/common-styles/` page -> Div with background and text 60% opacity

### PR DESCRIPTION
**Pull request by DAreRodz**
Modify `textColor` and `backgroundColor` processors to add colors with opacity.

Also, a function called `addAlpha` that adds an alpha channel to hex colors was added to `libraries.theme`.

Closes #3.

### CODE REVIEW

**Comment by DAreRodz** [comment]
> I'm wondering, why did you add the addAlpha function to libraries instead of just importing them statically?

Well, I did it that way just to tests `libraries` inside processors 😁, but if other packages don't need that function I guess we can move it to a file and import it statically.

**Comment by luisherranz** [comment]
> Well, I did it that way just to tests libraries inside processors 😁

Haha, nice.

Yes, if it's not necessary for other packages I guess it'd be better to import it statically to not confuse people looking at the code of this theme.

File: packages/frontity-org-theme/src/index.ts [hunk 1]
```diff
@@ -27,6 +27,15 @@ const frontityOrg: FrontityOrg = {
...
   libraries: {
+    theme: {
+      addAlpha: (hex, alpha) =>
+        `rgba(${hex
+          .match(/^#(.{2})(.{2})(.{2})$/)
+          .slice(1)
+          .map(value => Number(`0x${value}`))
+          .concat(alpha)
+          .join(", ")})`
+    },
```
Comment (luisherranz) [inline packages/frontity-org-theme/src/index.ts]: Move `addAlpha` to a static import.

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit bd380226f162c59bd02f0e0ddaf6fb2256334608

https://frontity-org-qsec8qntl.now.sh

**Pull request merged**